tasks/util.py

```diff
@@ -108,6 +108,13 @@ def classpath(obj):
     return classpath_ if classpath_ else 'tmp'
 
 
+def proper_task_name(task_id):
```

Comment:
Nitpicking: I would rename this to something like `unqualified_task_id`. The current name makes me think it's going to return the task name, but this also includes parameters and a hash.